### PR TITLE
get babel config from .babelrc  

### PR DESCRIPTION
**Pull request by jerexyz**

### CODE REVIEW

**Comment by jerexyz** [comment]
fix bug

**Comment by jerexyz** [comment]
https://github.com/vire/jest-vue-preprocessor/issues/4

**Comment by jerexyz** [comment]
why close this?

**Comment by vire** [comment]
Omg I

File: index.js [hunk 1]
```diff
@@ -1,16 +1,19 @@
+/* eslint-env node */
 const vueCompiler = require('vue-template-compiler');
 const vueNextCompiler = require('vue-template-es2015-compiler');
 const babelCore = require('babel-core');
+const findBabelConfig = require('find-babel-config');
 
 const transformBabel = src => {
+  const {config} = findBabelConfig.sync(process.cwd());
   const transformOptions = {
```
Comment (joezimjs) [inline index.js]: I don't see the point of keeping the default `transformOptions`.

Comment (jerexyz) [inline index.js]: yep，this can  be remove.  but considering that some people may not be configured.

Comment (joezimjs) [inline index.js]: If they're not configured, then they probably aren't using Babel

**Pull request merged**